mule-extensions-api/src/main/java/org/mule/runtime/extension/api/introspection/property/DisplayModelProperty.java

```diff
@@ -24,22 +24,46 @@
     private final int order;
     private final String groupName;
     private final String tabName;
+    private final String summary;
 
     public DisplayModelProperty(String displayName,
```

Comment:
This constructor is getting a bit over the top. Replace by an argless private constructor and add a builder